### PR TITLE
Reorganize navbar

### PR DESCRIPTION
**Pull request by antidipyramid**
## Overview

Condenses navbar items using dropdowns and fixes responsiveness of the bar.

### Checklist

- [x] PR has a descriptive enough title to be useful in changelogs

### Demo

navbar

### Notes

A section of `city_custom.css` was removed to change the navbar's breakpoint.

## Testing Instructions

 * Navigate to the site and verify that the navbar has been condensed and the dropdowns render correctly.
 * Verify that the navbar is responsive.

Handles #785

### CODE REVIEW

File: lametro/static/css/city_custom.css
Comment (fatima3558) [inline lametro/static/css/city_custom.css:348]: curious if you looked to see where else these classes were being used // if elements affected by these had their behavior changed by removing this styling?

Comment (antidipyramid) [reply lametro/static/css/city_custom.css:348]: I believe this block was just used to set the breakpoint for the navbar at a larger window size than bootstrap's default. Most likely because the navbar had so much on it.

It's a media rule plus only the navbar is styled here so I don't think anything else is affected. But if you saw any weird behavior, let me know 🙏🏾

Comment (antidipyramid) [reply lametro/static/css/city_custom.css:348]: @fatima3558 Anything you want me to modify before I get your approval?

**Review by hancush — APPROVED** [review]
@antidipyramid This LGTM!

**Pull request merged**
